pkgs/applications/science/logic/boolector/default.nix

```diff
@@ -40,22 +38,61 @@ stdenv.mkDerivation rec {
     cd ../src
     find . -iname '*.h' -exec cp --parents '{}' $dev/include \;
     rm -rf $dev/include/tests
-  '';
+  '' + (
+    if stdenv.isDarwin then ''
+      install_name_tool -change libboolector.dylib $lib/lib/libboolector.dylib $out/bin/boolector
+      install_name_tool -change libbtor2parser.dylib ${btor2tools.lib}/lib/libbtor2parser.dylib $out/bin/boolector
```

Comment:
This indicates that the install_name of btor2tools isn't set correctly, fixing the build there will elevate this problem for all projects that depend on it.